distribution/lib/Standard/Microsoft/0.0.0-dev/src/Internal/SQLServer_Dialect.enso

```diff
@@ -17,6 +17,7 @@ import Standard.Database.DB_Column.DB_Column
 import Standard.Database.DB_Table.DB_Table
 import Standard.Database.Dialect
 import Standard.Database.Internal.Base_Generator
+import Standard.Database.Internal.Base_Generator.lift_binary_op
```

Comment:
```suggestion
from Standard.Database.Internal.Base_Generator import lift_binary_op
```
Prefer this syntax for bring functions in but just a style thing.

Reply:
Done